vulnerabilities/importer.py

```diff
@@ -458,6 +460,22 @@ def clean_summary(self, summary):
         return summary
 
     def to_dict(self):
+        is_adv_v2 = (
+            self.severities
```

Comment:
What if references_v2, severities, affected_packages are empty

Reply:
In such case there is no way to differentiate between v1 and v2 and hence it shall be tarted as v1 ATM to prevent breaking v1 importers.